core/cmd/local_client_test.go

```diff
@@ -96,7 +96,7 @@ func TestClient_RunNodeShowsEnv(t *testing.T) {
 	assert.Contains(t, logs, "LOG_TO_DISK: true")
 	assert.Contains(t, logs, "MIN_INCOMING_CONFIRMATIONS: 1\\n")
 	assert.Contains(t, logs, "MIN_OUTGOING_CONFIRMATIONS: 6\\n")
-	assert.Contains(t, logs, "MINIMUM_CONTRACT_PAYMENT: 0.000000000000000100\\n")
+	assert.Contains(t, logs, "MINIMUM_CONTRACT_PAYMENT_LINK_JUELS: 0.000000000000000100\\n")
```

Comment:
This seems wrong... wouldn't it be 100 Juels?

Reply:
Ahh yep, the display functions will have to be updated.